src/Stratis.Features.FederatedPeg/TargetChain/MaturedBlocksSyncManager.cs

```diff
@@ -19,6 +19,8 @@ namespace Stratis.Features.FederatedPeg.TargetChain
     /// Handles block syncing between gateways on 2 chains. This node will request
     /// blocks from another chain to look for cross chain deposit transactions.
     /// </summary>
+    /// <remarks>Processes matured block deposits from the cirrus chain and creates instances of <see cref="ConversionRequest"/> which are
+    /// saved to <see cref="IConversionRequestRepository"/>.</remarks>
```

Comment:
Bear in mind that the creation of `ConversionRequest`s is something that was added onto this existing functionality. Conversion requests are obviously not used for normal cross-chain transfers.